[2024-10-29T22:10:40.913+0500] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-10-29T22:10:40.931+0500] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: text_processing_dag.count_a_in_files manual__2024-10-29T17:10:30.071735+00:00 [queued]>
[2024-10-29T22:10:40.941+0500] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: text_processing_dag.count_a_in_files manual__2024-10-29T17:10:30.071735+00:00 [queued]>
[2024-10-29T22:10:40.942+0500] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2024-10-29T22:10:40.961+0500] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): count_a_in_files> on 2024-10-29 17:10:30.071735+00:00
[2024-10-29T22:10:40.982+0500] {standard_task_runner.py:72} INFO - Started process 1492 to run task
[2024-10-29T22:10:40.994+0500] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'text_processing_dag', 'count_a_in_files', 'manual__2024-10-29T17:10:30.071735+00:00', '--job-id', '11', '--raw', '--subdir', 'DAGS_FOLDER/text_processing_dag.py', '--cfg-path', '/tmp/tmpfr9hlaff']
[2024-10-29T22:10:40.999+0500] {standard_task_runner.py:105} INFO - Job 11: Subtask count_a_in_files
[2024-10-29T22:10:41.073+0500] {task_command.py:467} INFO - Running <TaskInstance: text_processing_dag.count_a_in_files manual__2024-10-29T17:10:30.071735+00:00 [running]> on host Easyboom.
[2024-10-29T22:10:41.198+0500] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='text_processing_dag' AIRFLOW_CTX_TASK_ID='count_a_in_files' AIRFLOW_CTX_EXECUTION_DATE='2024-10-29T17:10:30.071735+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-10-29T17:10:30.071735+00:00'
[2024-10-29T22:10:41.200+0500] {taskinstance.py:731} INFO - ::endgroup::
[2024-10-29T22:10:43.340+0500] {python.py:240} INFO - Done. Returned value was: None
[2024-10-29T22:10:43.360+0500] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-10-29T22:10:43.361+0500] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=text_processing_dag, task_id=count_a_in_files, run_id=manual__2024-10-29T17:10:30.071735+00:00, execution_date=20241029T171030, start_date=20241029T171040, end_date=20241029T171043
[2024-10-29T22:10:43.391+0500] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-10-29T22:10:43.426+0500] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-10-29T22:10:43.430+0500] {local_task_job_runner.py:245} INFO - ::endgroup::
